docusaurus.config.js

```diff
@@ -69,11 +68,15 @@ const config = {
       }),
     ],
   ],
-
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       metadata: [{ name: "theme-color", content: "#815bb5" }],
+      markdown: {
```

Comment:
fix warning: 


[INFO] Starting the development server...
[WARNING] The `siteConfig.onBrokenMarkdownLinks` config option is deprecated and will be removed in Docusaurus v4.
Please migrate and move this option to `siteConfig.markdown.hooks.onBrokenMarkdownLinks` instead.
[SUCCESS] Docusaurus website is running at: http://localhost:3000/